app/models/tag.rb

```diff
@@ -150,7 +154,7 @@ def lists_that_do_not_include_list_item(list_item)
   end
 
   def can_be_archived?
-    level_two? && published?
+    raise NotImplementedError
```

Comment:
We shouldn't use this type of [Error](https://oleg0potapov.medium.com/ruby-notimplementederror-dont-use-it-dff1fd7228e5)... sorry I didn't get a chance to review this!